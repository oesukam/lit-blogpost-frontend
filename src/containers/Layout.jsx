import React from 'react';
import PropTypes from 'prop-types';
import TopNav from '../components/TopNav/TopNav';

const Layout = ({ childreen, match }) => (
  <div>
    <TopNav match={match} />
    <div className="main-content">{childreen}</div>
  </div>
);

Layout.propTypes = {
  childreen: PropTypes.any,
  match: PropTypes.any,
};

Layout.defaultProps = {
  childreen: '',
  match: { params: {} },
};

export default Layout;
