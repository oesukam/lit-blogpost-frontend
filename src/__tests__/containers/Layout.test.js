import React from 'react';
import { shallow } from 'enzyme';
import Layout from '../../containers/Layout';

let wrapper;

describe('Layout.jsx', () => {
  test('should render Layout.jx', () => {
    wrapper = shallow(<Layout />);
    expect(wrapper).toMatchSnapshot();
  });
});
