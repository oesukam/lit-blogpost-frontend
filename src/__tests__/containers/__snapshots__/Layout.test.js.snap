// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout.jsx should render Layout.jx 1`] = `ShallowWrapper {}`;
